### DISCUSSION TITLE
Supported filetypes?

### DISCUSSION THREAD

**Discussion by e-t-l** [gsantner/markor#1498] [Q&A] [answered]
Where would I be able to find a list of all filetypes that Markor can open?

**Comment by gsantner** [comment, marked as answer]
All plain textfiles. 

File extensions that are specifically registered are listed [here](https://github.com/gsantner/markor/blob/master/app/src/main/AndroidManifest.xml#L225) and (i.e. to distinguish markdown, plaintext, zim format etc) in the TextConverter.java class files.
